Documentation/installing-dependencies.rst

```diff
@@ -1,216 +1,293 @@
+-  SCOTCH
+-  ParHIP
 
-.. _installing_parmetis:
+The partitioning of SeisSol meshes with ParMETIS was tested in large simulations and is
+generally recommended for academic users.
+SCOTCH and ParHIP are free and open-source alternatives to ParMETIS and should be used by
+users from industry or for-profit institutions (cf. `ParMETIS license <https://github.com/KarypisLab/ParMETIS/blob/main/LICENSE>`_).
+A study comparing partition quality for SeisSol meshes can be found `here <https://home.in.tum.de/~schnelle/publications/bachelorsthesis-informatics-final.pdf>`_.
 
-Installing ParMetis
-~~~~~~~~~~~~~~~~~~~
+ParMETIS
+""""""""
 
+ParMETIS may be installed as follows:
 
 .. code-block:: bash
 
   wget https://ftp.mcs.anl.gov/pub/pdetools/spack-pkgs/parmetis-4.0.3.tar.gz
   tar -xvf parmetis-4.0.3.tar.gz
   cd parmetis-4.0.3
   #edit ./metis/include/metis.h IDXTYPEWIDTH to be 64 (default is 32).
-  make config cc=mpicc cxx=mpiCC prefix=$HOME 
+  make config cc=mpicc cxx=mpiCC prefix=$PREFIX 
   make install
-  cp build/Linux-x86_64/libmetis/libmetis.a $HOME/lib
-  cp metis/include/metis.h $HOME/include
+  cp build/Linux-x86_64/libmetis/libmetis.a $PREFIX/lib
+  cp metis/include/metis.h $PREFIX/include
   cd ..
 
 (Make sure $HOME/include contains metis.h and $HOME/lib contains
 libmetis.a. Otherwise, compile error: cannot find parmetis.)
 
+Other Software (optional)
+~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Installing ASAGI (Optional)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~
+netCDF
+""""""
 
-See section :ref:`Installing ASAGI <installing_ASAGI>`.
-
-.. _compiling-seissol:
-
-Installing easi
-~~~~~~~~~~~~~~~
-
-Follow the `installation instructions <https://easyinit.readthedocs.io/en/latest/getting_started.html>`_.
-
-
-Installing GemmForge, ChainForge (for GPUs)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-.. _gemmforge_installation:
+NetCDF is needed for convergence tests, as these use periodic boundary conditions—and such are not supported by the PUML mesh format.
+Also, point sources utilize the netCDF backend.
+Once again, if you do not have it installed (sometimes it comes bundled with HDF5), you may do so manually.
 
 .. code-block:: bash
 
-   pip3 install --user git+https://github.com/SeisSol/gemmforge.git
-   pip3 install --user git+https://github.com/SeisSol/chainforge.git
+  wget https://syncandshare.lrz.de/dl/fiJNAokgbe2vNU66Ru17DAjT/netcdf-4.6.1.tar.gz
+  tar -xaf netcdf-4.6.1.tar.gz
+  cd netcdf-4.6.1
+  CFLAGS="-fPIC ${CFLAGS}" CC=h5pcc ./configure --enable-shared=no --prefix=$HOME --disable-dap
+  #NOTE: Check for this line to make sure netCDF is built with parallel I/O: 
+  #"checking whether parallel I/O features are to be included... yes" This line comes at the very end (last 50 lines of configure run)!
+  make -j8
+  make install
+  cd ..
 
-Installing SYCL (for GPUs)
-~~~~~~~~~~~~~~~~~~~~~~~~~~
+ASAGI
+"""""
 
-See section :ref:`Installing SYCL <installing_SYCL>`.
+See section :ref:`Installing ASAGI <installing_ASAGI>`.
```

Comment:
Add: `If you want to use SeisSol with ASAGI, you need to install `easi` with ASAGI support.`